shell/layouts/default.vue

```diff
@@ -237,7 +237,7 @@ export default {
     <div
       v-if="managementReady"
       class="dashboard-content"
-      :class="{[pinClass]: true}"
+      :class="{[pinClass]: true, 'dashboard-padding-left': isRancherInHarvester || isMultiCluster || !isSingleProduct}"
```

Comment:
it looks like these all match the if statement for `TopLevelMenu`. i think it's a sound idea to add padding using the same condition.

however they should be done in a common way to avoid regression. as `isRancherInHarvester`, `isMultiCluster` and `isSingleProduct` all come from the root store, suggest adding a `showTopLevelMenu` getter there which is used in the three new places and in the header for TopLevelMenu